src/Explorer/Graph/GraphExplorerComponent/GraphData.ts

```diff
@@ -413,14 +413,14 @@ export class GraphData<V extends GremlinVertex, E extends GremlinEdge> {
    * @param node
    * @param prop
    */
-  public static getNodePropValue(node: D3Node, prop: string): string | number | boolean {
+  public static getNodePropValue(node: D3Node, prop: string): undefined | string | number | boolean {
     if (node.hasOwnProperty(prop)) {
       return (node as any)[prop];
     }
 
     // This is DocDB specific
-    if (node.hasOwnProperty("properties") && node.properties.hasOwnProperty(prop)) {
-      return node.properties[prop][0]["value"];
+    if (node.hasOwnProperty("properties") && node.properties!.hasOwnProperty(prop)) {
```

Comment:
send in a different PR and check with Steve and Laurent if we prefer the following style

    if (node && node.properties && node.properties.prop) {

Reply:
I chenged to node && node.properties && node.properties.prop &&, which has all properties in ndoe or not, but not 100% sure, will ask Steve and laurent

Reply:
I would opt for: `if (node?.properties?.prop) {`